docs/development.md

```diff
@@ -685,4 +685,42 @@ You just need to set `TYK_OPERATOR_PRELOAD_IMAGES` environment variable to true
 
 ## Highlights
 
-1. The Tyk-Operator uses the [finalizer](https://book.kubebuilder.io/reference/using-finalizers.html) pattern for deleting CRs from the cluster.
\ No newline at end of file
+1. The Tyk-Operator uses the [finalizer](https://book.kubebuilder.io/reference/using-finalizers.html) pattern for deleting CRs from the cluster.
+
+
+
+
+## IDE Integration
```

Comment:
I believe that the IDE Integration section shouldn't belong to development documentation. Although users do not develop Operator, they should somehow be aware of IDE Integration. It might be better to keep it in the main Readme